src/include/btree_inline.h

```diff
@@ -1737,6 +1737,7 @@ __wt_leaf_page_can_split(WT_SESSION_IMPL *session, WT_PAGE *page)
     WT_INSERT_HEAD *ins_head;
     size_t size;
     int count;
+    size_t mem_split_threshold = 0;
```

Comment:
We can remove that line. Initializing is good though, removing the `size_t` would be enough too.